client/src/components/Production/ProductionTable.jsx

```diff
@@ -0,0 +1,95 @@
+import { Table, Thead, Tbody, Tr, Th, TableCaption, Heading } from '@chakra-ui/react';
+import { StyledTableRow, StyledTableCell } from 'components/common/Table.jsx';
+import ProductionModal from 'components/Production/ProductionModal.jsx';
+import { TablePagination } from '@material-ui/core';
+import { Fragment } from 'react';
+import Loader from 'components/common/Loader';
+import { NoResultImage } from 'components/common/Image.jsx';
+import useProductionTable from 'hooks/useProductionTable';
+
+const ProductionTable = () => {
+  const {
+    isLoadingProduction,
+    isSuccessProduction,
+    dataProduction,
+    isLoadingPart,
+    isSuccessPart,
+    isLoadingBike,
+    isSuccessBike,
+    page,
+    rowsPerPage,
+    findEntity,
+    handleChangePage,
+    handleChangeRowsPerPage,
+    formatDate,
+  } = useProductionTable();
+
+  if (isLoadingProduction && isLoadingBike && isLoadingPart) {
+    return <Loader />;
+  }
```

Comment:
If one of these is done loading, but the other two are still loading, the loader will no longer be displayed. Is that the intended behaviour?